src/components/common/OptionsMenu.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { Box } from '@mui/material';
+import { Button, DropdownItem, Popover, Tooltip } from '@terraware/web-components';
+import strings from 'src/strings';
+
+export type OptionsMenuProps = {
+  optionItems: DropdownItem[];
+  onOptionItemClick?: (optionItem: DropdownItem) => void;
+  size?: 'medium' | 'small';
+};
+
+export default function OptionsMenu({ optionItems, onOptionItemClick, size }: OptionsMenuProps): JSX.Element {
```

Comment:
Extracted the OptionsMenu into a separate component since it was being implemented in every use-case.